testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -762,8 +762,8 @@ public void testTrinoWritingDataWithObjectStorageLocationProvider(StorageFormat
             "with€euro",
             "with non-ascii ąęłóść Θ Φ Δ",
             "with👨‍🏭combining character",
-            " 👨‍🏭",
-            "👨‍🏭 ");
+            "👨‍🏭 ",
+            " 👨‍🏭");
```

Comment:
how is this related?

Reply:
For some reason the style check plugin doesn't like the space at the end.

Reply:
Did you find a bug in style check plugin?
I don't think it's OK to forbid a space at the end of the string literal, even after an emoji.

Reply:
Yeah, I think it's just a bug. Fixing it would require another turnaround of migrations. Do you think it is worth an another migration effort?

Reply:
No, let's report the bug and downgrade the plugin for now.

Reply:
I think we need that version of the plugin for Java 17

Reply:
https://github.com/airlift/airbase/commit/831369f2be2c8bfe4b585e30a0082bbf01a2ef68

Reply:
Is it modernizer's fault, or checkstyle's?

Reply:
It's a checkstyle violation

```
[INFO] There is 1 error reported by Checkstyle 9.1 with checkstyle/airbase-checks.xml ruleset.
[ERROR] src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java:[766,20] (whitespace) NoWhitespaceBefore: ';' is preceded with whitespace.
```

The checkstyle version got updated to fix an another bug: https://github.com/airlift/airbase/commit/69c1b58692afd618f9f263555338b2340f881f12

It looks like checkstyle 9.2 already has a fix for this problem: https://github.com/checkstyle/checkstyle/issues/10920

Reply:
https://github.com/airlift/airbase/pull/288